tests/common/include/tests/common/tools/intersectors/helix_cylinder_intersector.hpp

```diff
@@ -103,7 +103,7 @@ struct helix_cylinder_intersector {
         intersection_type &is = ret[0];
         const point3 helix_pos = h.pos(s);
 
-        is.path = getter::norm(helix_pos);
+        is.path = s;
```

Comment:
But ```s``` is the distance along the helix, not the distance from the current helix position to the surface...

Reply:
I was using it for different purpose to rewrite the path correction test - I found that we can test the path correction without tracking geometry (sry) and stepper but just with helix and helix intersector. Techincally, I had to generate the transport jacobian from `helix` using the `is.path` from the helix intersector.

BTW, `getter::norm(helix_pos)` is also not right for the original purpose - it should have been like `getter::norm(helix_pos- helix._pos)`

Reply:
The helix in the particle gun starts in the origin and is not getting propagated, but, yes, you are right, of course, this line is incorrect anyway :) I agree, putting it to the helix path makes more sense and I have to solve the intersector comparison differently, if that is even actually possible